src/frontend/context/OppgaverContext.tsx

```diff
@@ -282,7 +283,31 @@ const [OppgaverProvider, useOppgaver] = createUseContext(() => {
               )
             : [];
 
-    const fordelOppgave = (oppgave: IOppgave, saksbehandler: string): Promise<Ressurs<string>> => {
+    const harLøpendeSakIInfotrygd = async (bruker: string): Promise<boolean | undefined> => {
+        return await request<{ ident: string }, { harLøpendeSak: boolean }>({
+            method: 'POST',
+            data: { ident: bruker },
+            url: '/familie-ba-sak/api/infotrygd/har-lopende-sak',
+        })
+            .then((res: Ressurs<{ harLøpendeSak: boolean }>) => {
+                if (res.status !== RessursStatus.SUKSESS) {
+                    return undefined;
+                } else if (res.data.harLøpendeSak) {
```

Comment:
Kan man returnere harLøpendeSak direkte her?
```
else {
    return res.data.harLøpendeSak
}
```

Reply:
Ikke helt. Denne funksjonen returnerer tre verdier: boolean og undefined, der undefined brukes for å signalisere en feil. Man kan bruke en resultattype, og kanskje jeg burde lage en. Vurderte først Ressurs, men den er litt tungvekter i forhold til funksjonen.

Reply:
Ved nærmere ettertanke blir det bedre å sende ressursen videre - det hele ble mer konsist.